pkg/jsonld/loader.go

```diff
@@ -88,6 +90,10 @@ var contextDocuments = []jsonld.ContextDocument{
 		URL:     "https://trustbloc.github.io/context/vc/examples/credit-card-v1.jsonld",
 		Content: creditCardV1Vocab,
 	},
+	{
+		URL:     "https://trustbloc.github.io/context/vc/examples/credit-score-v1.jsonld",
```

Comment:
Note: This would be removed once we have the capability to dynamically add json-ld contexts. Till then this is not scalable and whenever we have new use-case in sandbox, we need to update edge-adapter codebase to add the context to this list.

Reply:
exactly